pkg/pool-weighted/test/ManagedPool.test.ts

```diff
@@ -682,12 +682,53 @@ describe('ManagedPool', function () {
         const receipt = await pool.instance.updateCachedProtocolSwapFeePercentage();
 
         // Real Vault will return a zero protocol fee
-        expectEvent.inReceipt(await receipt.wait(), 'ProtocolSwapFeeCacheUpdated', {
+        expectEvent.inReceipt(await receipt.wait(), 'CachedProtocolSwapFeePercentageUpdated', {
           protocolSwapFeePercentage: 0,
         });
       });
     });
 
+    describe('Non-delegated protocol fees', () => {
```

Comment:
```suggestion
    describe('non-delegated protocol fees', () => {
```

Reply:
Also, these tests should be in a separate file, and we should remove the duplcate tests that StablePhantomPool has.